nextcloudappstore/core/forms.py

```diff
@@ -50,27 +51,40 @@ class AppRegisterForm(Form):
                               'openssl base64</b>'))
     safe_help_fields = ['certificate', 'signature']
 
+def get_languages_local(language=None):
+    if language:
+        languages = list(language)
+    else:
+        languages = [ l[0] for l in settings.LANGUAGES]
+
+    language_infos = list()
+    for l in languages:
```

Comment:
but I can't come up with a list comprehension without calling get_language_info(l) twice for each language, which I don't think is a good idea either.